app/helpers/commodities_helper.rb

```diff
@@ -97,10 +97,10 @@ def tree_node(main_commodity, commodities, depth)
     if deeper_node.present? && deeper_node.number_indents < main_commodity.number_indents
       tag.ul do
         tag.li do
-          tree_code(deeper_node.code.gsub(/[0]{2}+$/, '')) +
+          tree_code(deeper_node.code.gsub(/0{2}+$/, '')) +
             tag.p(deeper_node.to_s.html_safe) +
             # if deeper_node.producline_suffix == '80'
-            #   tree_code(deeper_node.code.gsub(/[0]{2}+$/, ''))
+            #   tree_code(deeper_node.code.gsub(/0{2}+$/, ''))
             # end +
```

Comment:
Do we still need this code?

Reply:
Probably not, it's been commented out for ten months now.